beacon_node/beacon_chain/src/block_verification_types.rs

```diff
@@ -145,13 +152,16 @@ impl<E: EthSpec> RpcBlock<E> {
         Ok(Self {
             block_root,
             block: inner,
+            // Block is before PeerDAS
+            custody_columns_count: 0,
         })
     }
 
```

Comment:
might be worth having consistent terminology for:
1. Blocks
2. Post-Deneb Blocks (blocks with blobs)
3. Post-PeerDAS Blocks (blocks with custody columns)

Not sure what terminology that would be, but if we don't already have one / can't think of one, we should at least rename the above `new` function to something like `new_with_blobs` or something.

Reply:
I want to get rid of RpcBlock completely, so this assignment here will become irrelevant.